packages/std/src/iterator.rs

```diff
@@ -1,8 +1,8 @@
 use crate::errors::StdError;
 use std::convert::TryFrom;
 
-/// A Key-Value pair, returned from our iterators
-pub type Pair<V = Vec<u8>> = (Vec<u8>, V);
+/// A pair of values, returned from our iterators
+pub type Pair<A = Vec<u8>, B = Vec<u8>> = (A, B);
```

Comment:
I am strongly against using A and B here. This will only add to more dev confusion.

At least having some type hints would be good. I commented on the issue to the same effect.